spec/factory_trace/integration_tests/expected.txt

```diff
@@ -1,14 +1,16 @@
-total number of unique used factories & traits: 0
-total number of unique unused factories & traits: 12
-unused factory user => spec/factories.rb:19
-unused trait with_phone of factory user => spec/factories.rb:22
-unused factory user_with_defaults => spec/factories.rb:26
-unused factory admin => spec/factories.rb:29
-unused trait with_email of factory admin => spec/factories.rb:30
-unused trait combination of factory admin => spec/factories.rb:34
-unused factory manager => spec/factories.rb:40
-unused factory company => spec/factories.rb:44
-unused trait with_manager of factory company => spec/factories.rb:45
-unused factory article, post => spec/factories.rb:50
-unused factory comment => spec/factories.rb:52
-unused global trait with_address => spec/factories.rb:56
+total number of unique used factories & traits: 2
+total number of unique unused factories & traits: 14
```

Comment:
**Question:** Could you please remind me why this has changed? And why we are okay with this?

Reply:
Yes, sure. I have added a new factory `:task` in the `spec/factories.rb` file. Inside this factory, I use `traits_for_enum` to declare 3 traits (queued, started, and finished) based on the status attribute.
One of that trait I'm using inside `spec/factory_trace/integration_tests/empty_spec.rb`. This means that during integration we start using 1 factory + 1 trait and we do not use the remaining 2 traits.

Reply:
I see, understand then.